src/python/pants/backend/python/goals/export.py

```diff
@@ -86,39 +85,28 @@ async def export_virtualenv(
             request.root_python_targets, python_setup
         ) or InterpreterConstraints(python_setup.interpreter_constraints)
 
-    min_interpreter = interpreter_constraints.snap_to_minimum(python_setup.interpreter_universe)
```

Comment:
This interpreter was not the one being used in practice! This was the legacy of the previous implementation.